net/advanced-operations/compare-pdf-documents/_index.md

```diff
@@ -90,4 +94,128 @@ The comparison results generated by these snippets are PDF documents that you ca
+
+With Aspose.PDF for .NET, it's possible to compare documents and pages and output the comparison result to a PDF document or image file.
+
+You can set the following class properties:
+
+- Resolution - resolution in DPI units for output images, as well as for images generated during the comparison.
+- Color - the color of change marks.
+- Threshold - change threshold in percent. The default value is zero. Setting a value other than zero allows you to ignore graphic changes that are insignificant to you.
+
+The class has a method that allows you to get page image differences in a form suitable for further processing: **ImagesDifference GetDifference(Page page1, Page page2)**.
+
+This method returns an object of the [ImagesDifference](https://reference.aspose.com/pdf/net/aspose.pdf.comparison.graphicalcomparison/imagesdifference/) class, which contains an image of the first page being compared and an array of differences. The array of differences and the original image has the **RGB24bpp** pixel format.
+
+ImagesDifference allows you to generate a different image and get an image of the second page being compared by adding an array of differences to the original image. To do this, use the **ImagesDifference.GetDestinationImage and ImagesDifference.DifferenceToImage** methods.
+
+### Compare PDF with GetDifference method
+
+The provided code defines a method [GetDifference](https://reference.aspose.com/pdf/net/aspose.pdf.comparison.graphicalcomparison/imagesdifference/#methods) that compares two PDF documents and generates visual representations of the differences between them.
+
+This method compares the first pages of two PDF files and generates two PNG images:
+
+- One image (diffPngFilePath) highlights the differences between the pages in red.
+- The other image (destPngFilePath) is a visual representation of the destination (second) PDF page.
+
+This process can be useful for visually comparing changes or differences between two versions of a document.
+
+```cs
+
+    string doc1Path = "";
+    string doc2Path = "";
+    string destPngFilePath = "";
+    string diffPngFilePath = "";
+
+    using (Document doc1 = new Document(doc1Path), doc2 = new Document(doc2Path))
+    {
+        GraphicalPdfComparer comparer = new GraphicalPdfComparer();
+        using (ImagesDifference imagesDifference = comparer.GetDifference(doc1.Pages[1], doc2.Pages[1]))
+        {
+
+            using (Bitmap diffImg = imagesDifference.DifferenceToImage(Color.Red, Color.White))
+            {
+                diffImg.Save(diffPngFilePath);
+            }
+
+            using (Bitmap destImg = imagesDifference.GetDestinationImage())
+            {
+                destImg.Save(destPngFilePath);
+            }
+        }
+    }
+```
+
+The provided code snippet used the [Aspose.PDF.Drawing](https://docs.aspose.com/pdf/net/drawing/) library and compares two PDF documents visually.  
```

Comment:
Could you move this to the top of the article? Please mention that all comparing tools are also available in PDF.Drawing lib.